File: src/main/java/tech/gsoftware/macrox/data/Item.java
package tech.gsoftware.macrox.data;

import java.util.List;

public class Item {
    private String name;
    private String path;
    private List<Integer> keys;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public List<Integer> getKeys() {
        return keys;
    }

    public void setKeys(List<Integer> keys) {
        this.keys = keys;
    }
}

File: src/main/java/tech/gsoftware/macrox/data/Item.kt
package tech.gsoftware.macrox.data

/**
 * Item.kt
 * @author          woz07
 * @description     Data class regarding info that gets displayed to ui
 */

data class Item(
    var name:   String? = null,
    var path:   String? = null,
    var keys:   List<Int>? = null,
    var active: Boolean? = null
)
